android/assets/jsons/translations/French.properties

```diff
@@ -3715,27 +3391,20 @@ Heal adjacent units for an additional 15 HP per turn = Soigne les unités adjace
 
 In your first couple of turns, you will have very little options, but as your civilization grows, so do the number of things requiring your attention = Lors de vos premiers tours,\n vous aurez peu de choix.\n Mais plus votre civilisation grandira, \n plus vous aurez d'éléments à gérer.
 
 Each turn, the culture you gain from all your cities is added to your Civilization's culture.\nWhen you have enough culture, you may pick a Social Policy, each one giving you a certain bonus. = A chaque tour, la culture que vous gagnez grâce à vos\n  villes est ajoutée à votre total de culture.\nLorsque vous avez suffisament de culture, vous pouvez choisir\n  une Doctrine, chaque culture vous donnera un bonus différent.
 The policies are organized into branches, with each branch providing a bonus ability when all policies in the branch have been adopted. = Les Doctrines sont organisées en arbre, chaque\n  arbre donne un bonus supplémentaire lorsque\n  toutes les doctrines de l'arbre ont été adoptées.
 With each policy adopted, and with each city built,\n  the cost of adopting another policy rises - so choose wisely! = Pour chaque doctrine adoptée et chaque ville construite,\n le coût pour adopter une nouvelle doctrine augmente.
 
- # Requires translation!
-Once a city has gathered enough Culture, it will expand into a neighboring tile.\nYou have no control over the tile it will expand into, but tiles with resources and higher yields are prioritized. = 
- # Requires translation!
-Each additional tile will require more culture, but generally your first cities will eventually expand to a wide tile range. = 
- # Requires translation!
-Although your city will keep expanding forever, your citizens can only work 3 tiles away from city center.\nThis should be taken into account when placing new cities. = 
+Once a city has gathered enough Culture, it will expand into a neighboring tile.\nYou have no control over the tile it will expand into, but tiles with resources and higher yields are prioritized. = Lorsque une ville a accumulé assez de culture, elle prendra possession d'une des cases adjacentes. Vous n'avez aucun contrôle sur la case qui sera choisie, mais celles disposant de ressources particulières ou simplement ayant le plus de ressources seront favorisées.
```

Comment:
I noticed you've removed a lot of capital letters, but while it's grammatically correct, it might not always be a good idea. For instance here, "Culture" refers to an amount of an in-game value. When the word refers to such concepts, it might be clearer to capitalize them (as in the original english) to easily identify them.

Reply:
I confess I didn't even noticed those capital-or-not letters in the game, except when there was some inconsistency about it. Ok, while going through all of it again, I will make it always with capital letters

Reply:
To be honest, it's just something I'm assuming, it would be better if the game would highlight them (or show them in a specific color) so it's more obvious..